### PR TITLE
New - Cyber Security team page

### PR DESCRIPTION
**Pull request by soupdragon99**
Content outlining the Cyber Security team's responsibilities.

### CODE REVIEW

**Comment by soupdragon99** [comment]
currently being reviewed by Martin Sivorn.

**Comment by soupdragon99** [comment]
Martin on annual leave - to be reviewed.

**Comment by soupdragon99** [comment]
Martin reviewing content

File: source/index.html.md.erb [hunk 1]
```diff
@@ -57,11 +57,12 @@ These standards will enable us to maintain a consistent operating environment ac
 - [how to store and query logs](standards/logging.html)
 - [how to do penetration tests](standards/how-to-do-penetration-tests.html)
 - [how to manage access to your third-party service accounts](standards/accounts-with-third-parties.html)
+- [Use configuration management](standards/configuration-management.html)
```
Comment (rhowe-gds) [inline source/index.html.md.erb]: Is this configuration management line an artifact of a merge conflict? It seems like it shouldn't be part of this PR

Comment (soupdragon99) [reply source/index.html.md.erb]: Yes, I think it's due to a merge conflict because this PR has been open for a while.

**Comment by soupdragon99** [comment]
meeting scheduled with Martin 11th July to discuss.

**Comment by soupdragon99** [comment]
Updated Cyber Security team overview based on recent changes.

**Comment by soupdragon99** [comment]
Closing PR. The team structure is changing and will not be finalised until later this quarter. Will log fresh PR when we have the new team structure and content.
